app/views/search/results.html.erb

```diff
@@ -39,7 +39,11 @@
     <div class="col3q wrap-results">
       <% if @results.present? %>
         <ul id="results" class="list-unbulleted">
-          <%= render(partial: 'search/result', collection: @results) %>
+          <% if Flipflop.enabled?(:gdt) %>
```

Comment:
I have moderate concerns about forking `results` entirely rather than determining which features are geo specific and which are appropriate for all timdex-uis. The concern stems from the fact we'll eventually need to figure this out and I'm not sure if we'll ever have more insight into the differences/similarities than we do now.

This isn't blocking, but it is requesting a bit of context as to why this path was chosen.

Reply:
They're similar enough that I did consider using the same partial with a handful of conditions. This decision was informed partly by my preference for smaller partials, and partly by my assumption that this approach would be preferable for the record view. (The logic would get more complex in that case, and I think more thought needs to be given to what fields we want in non-GDT records before we assume that they should be close to identical.)

I also have questions about some questions about the certain details that I don't think we can answer yet (e.g., whether 'summary' be in the result view for all record types). But, I understand your point that we may not gain further insight into that for a long time. I also think it would be easier to reconcile the differences between GDT and non-GDT views if we throw them both into the same file. That goes for the record view, too: while a single partial will be messier now, it could make things easier on us later.

I'm fairly ambivalent on this one, and happy to go the opposite route if you have a strong preference.

Reply:
Thanks for the context. I'm okay with going with your decision here to use the separate partials to make each partial cleaner and deal with this all later. As the only source that has gotten actual attention is GIS, I've come around to agreeing that we'll need more eyes on the non-GIS records to not just be guessing as to what should be the same.